client/package.json

```diff
@@ -19,6 +19,7 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
+    "apollo-client-preset": "1.0.8",
```

Comment:
this package has been deprecated. we should not use this. Rather use `apollo-boost` so we don't have issues in the future